rust/README.md

```diff
@@ -130,26 +131,25 @@ That means that the collar has been pushed buffered measurements to the app.
 Write down the pet Owner ID (ID is something after the `#` sign without trailing spaces).
 To start REST API service execute the following in the separate terminal:
 
-    $ HOST_IP=$(hostname -I | awk '{print $1}')
-    $ cargo run -- --hosts $HOST_IP
+    $ cargo run server 
 
 Expected output:
 
     2021-12-24T00:32:48Z INFO  care_pet::db] Connecting to 172.nnn.nnn.nnn
     [2021-12-24T00:32:48Z WARN  rocket::config::config] 🔧 Configured for debug.
-    [2021-12-24T00:32:48Z WARN  rocket::config::config] address: 127.0.0.1
+    [2021-12-24T00:32:48Z WARN  rocket::config::config] address: 0.0.0.0
     [2021-12-24T00:32:48Z WARN  rocket::config::config] port: 8000
     [2021-12-24T00:32:48Z WARN  rocket::config::config] workers: 16
     [2021-12-24T00:32:48Z WARN  rocket::config::config] ident: Rocket
     [2021-12-24T00:32:48Z WARN  rocket::config::config] keep-alive: 5s
     ...
 
-Now you can open `http://127.0.0.1:8000/` in the browser or send an HTTP request from the CLI:
+Now you can open `http://0.0.0.0:8000/` in the browser or send an HTTP request from the CLI:
```

Comment:
Why change to 0.0.0.0?

Reply:
I'm used to 0.0.0.0, more a preference on that. But just saw that I replaced ALL the occurrences in the rust docs LOL. Let me rollback this.